### DISCUSSION TITLE
[Art competition] Static Analyzer

### DISCUSSION THREAD

**Discussion by jinho-choi123** [prosyslab-classroom/cs348-information-security#428] [Art competition]
![_ee5cc526-3b87-44d7-93c8-fa1b52d2923f](https://github.com/prosyslab-classroom/cs348-information-security/assets/61547655/da6e846d-350b-4e94-999d-150ee3a14a2d)

## Short Description
For me, [Static Analyzer] means [realistic style painting. Soldiers are demining at the field using metal detectors.They find many mines, but few mines are left, hidden.].

## Long Description (up to 3 sentences)

Each soldier(static analyzer) is finding mine(bug) before somebody steps on it(program crash). Metal detector is used for abstraction, which abstracts mine as metal. If metal detector alarms, there may be a mine or it could be a false alarm that detected garbage metals.

**Comment by KihongHeo** [comment]
Please submit separately.
